build.gradle

```diff
@@ -322,7 +324,7 @@ task all(dependsOn: ['build', 'slowtest', 'javadoc', 'buildNsis', 'allReports'])
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.5'
+    gradleVersion = '4.0'
```

Comment:
lets revert that for now and make this change separate. Its easier to figure out what broke things with smaller more concrete changes,